halo2_gadgets/src/utilities/lookup_range_check.rs

```diff
@@ -49,22 +49,29 @@ impl<F: PrimeFieldBits> RangeConstrained<F, AssignedCell<F, F>> {
             .map(|inner| Self {
                 inner,
                 num_bits,
-                _phantom: PhantomData::default(),
+                _phantom: PhantomData,
             })
     }
 }
 
+// FIXME: add a proper doc
+/// ZsaExtension
+#[derive(Eq, PartialEq, Debug, Clone, Copy)]
+pub struct ZsaExtension {
```

Comment:
"ZsaExtension" --> "lookup_extension" or "lookup_decomposition_extension". The updated version extend the lookup range check to include the optimized short range check on 4 and 5 bits.